src/coreclr/jit/inline.h

```diff
@@ -702,9 +702,10 @@ struct InlineInfo
     bool hasSIMDTypeArgLocalOrReturn;
 #endif // FEATURE_SIMD
 
-    GenTreeCall* iciCall;  // The GT_CALL node to be inlined.
-    Statement*   iciStmt;  // The statement iciCall is in.
-    BasicBlock*  iciBlock; // The basic block iciStmt is in.
+    GenTreeCall* iciCall;          // The GT_CALL node to be inlined.
+    GenTree*     iciCallInstParam; // Cached InstParam arg (if exists) in iciCall
```

Comment:
Super nit: Put this next to the fields that relate to the arguments further above?